### DISCUSSION TITLE
UPPAAL STRATEGO 10 Beta 10 asks for a license key

### DISCUSSION THREAD

**Discussion by AstridHornBrorholt** [UPPAALModelChecker/UPPAAL-Meta#132] [Q&A] [answered]
Hello :-)
After installing Beta 10 on my new device, I am prompted for a license when I try to connect to the bundled server.

![image](https://user-images.githubusercontent.com/11016262/188452851-68cd7af1-4eff-41d1-83f1-8f72a92ebc14.png)

What is the test license I should try here?

**Comment by yrke** [comment, marked as answer]
Hi

Please write to marius@cs.aau.dk to get the license for the latest beta release, 

Regards, Kenneth

**Reply by AstridHornBrorholt** [reply]
Thanks!

**Comment by mikucionisaau** [comment]
The license keys can now be acquired here:
https://uppaal.veriaal.dk/register.html
